### PR TITLE
add docker compose and .env file for redis/mongo host

### PR DESCRIPTION
**Pull request by patrick-zhang**
Issue 6100 https://github.com/NodeBB/NodeBB/issues/6100
Add docker compose file for easy local development

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# https://docs.docker.com/compose/compose-file/
+
+version: '3'
+services:
+  node_bb:
```
Comment (julianlam) [inline docker-compose.yml]: Can you replace all references of `node_bb` to `nodebb` please?

**Comment by julianlam** [comment]
@BenLubar do we need this file?

File: docker-compose.yml (hunk 1 above)
Comment (patrick-zhang) [reply docker-compose.yml]: Done~

**Comment by henriquegomes6** [comment]
Please approve the pull request.
People need this to test, as installing everything in the documentation will take time and may not work on the first try.

**Pull request closed without merging**